prime-router/settings/put-local-settings.py

```diff
@@ -18,7 +18,7 @@
 org_file = args.org_file
 
 base_url = f"{protocol}://{host}:7071/api"
-headers = {"Authorization": "bearer xyz", "Content-Type": "application/json"}
+headers = {"Authorization": "bearer xyz", "Content-Type": "application/json", "SettingsSetup": "true"}
```

Comment:
Sending this header makes it so the local environment can set up the Settings database without authentication, and we test authentication in the actual application where appropriate.
I check against the PRIME_ENVIRONMENT and this header in the Okta Authentication class.
Open to other suggestions :laughing:

Reply:
I get what you are doing here:   Currently, Rick's code (that you have now moved into OktaAuthentication.kt) turns off all Okta authentication if PRIME_ENVIRONMENT is 'local', and you want to instead default to having Okta auth "on" locally, unless this header is sent. 

A couple thoughts:  Not just for settings setup, but for almost all our endpoints Its super convenient (or maybe lazy) to not have to authenticate when local.  

Also, while I realize this feature sort of actually increases security, I'm worried this makes it just a bit easier to mistakenly mess up the code in OktaAuthentication and create a breach.

Are there other options?   For example, maybe if PRIME_ENVIRONMENT is 'local' it uses Okta, but if PRIME_ENVIRONMENT is 'local_noauth' it doesn't?   Or maybe a second env variable?  other ideas?